src/features/ModelPlan/ModelToOperations/_components/MatrixTable/index.tsx

```diff
@@ -257,7 +257,7 @@ const MTOTable = ({
   }, [formattedData]);
 
   const totalPages = useMemo(() => {
-    return Math.ceil(itemLength / itemsPerPage);
+    return Math.floor(itemLength / itemsPerPage);
```

Comment:
Any specific use to cause this change? It didnt seem affecting this pr

Reply:
I think that snuck in when I was troubleshooting.  It doesn't affect anything but I'll revert it.